src/devices/src/virtio/vsock/csm/connection.rs

```diff
@@ -622,6 +622,7 @@ where
     }
 }
 
+/*
```

Comment:
I don't understand why these tests don't work with vsock-over-polly. The internal vsock implementation barely changes.

Reply:
More or less all the unit tests used a vsock `TestContext` which in turn used an `EpollHandlerContext` so they did not work by default. After some tinkering I think I've fixed them, but they had to be commented in the original version of the PR.